demo_mode.gemspec

```diff
@@ -20,6 +20,7 @@ Gem::Specification.new do |s|
 
   s.add_dependency 'rails', '>= 5.2'
   s.add_dependency 'sprockets-rails'
+  s.add_dependency 'sqlite3'
```

Comment:
This should go back in the Gemfile or under an `add_development_dependency` -- we don't need consuming apps to install it if they use a different DB for their models.

Reply:
thanks -- I moved it here because rubocop kept complaining it was a development dependency but I wasn't sure about that.